app/views/courses/apprenticeship/_form_fields.html.erb

```diff
@@ -0,0 +1,27 @@
+<div
+  class="<%= cns("govuk-form-group", "govuk-form-group--error": @errors && @errors[:program_type]&.any?) %>"
+  data-qa="course__program_type">
+
+  <fieldset class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+      <h1 class="govuk-fieldset__heading">
+        Is this a teaching apprenticeship?
+      </h1>
+    </legend>
+
+    <%= render "shared/error_messages", field: :program_type if @errors %>
+    <div class="govuk-radios" data-module="radios">
+      <% @course.meta['edit_options']['program_type'].each do |value| %>
+        <div class="govuk-radios__item">
+          <%= form.radio_button :program_type,
+                  value,
+                  class: 'govuk-radios__input' %>
+          <%= form.label :program_type,
+                  t("edit_options.apprenticeship.#{value}.label"),
```

Comment:
https://github.com/DFE-Digital/manage-courses-frontend/blob/04faad48968f72f8b19d40e4e20b66e33faea47e/config/locales/en.yml#L111

Program type can those as well, should the new course page be hardcoded... I do it on https://github.com/DFE-Digital/manage-courses-frontend/pull/579/files